lib/decompress_seq.ml

```diff
@@ -34,7 +34,7 @@ module MList = struct
     let start = ref Nil in
     let chunk_size = ref 8 in
     (* XXX: fill the list. pref: tail-reference from previous node *)
-    let prev, cur = ref start, ref Nil in
+    let prev, cur = (ref start, ref Nil) in
```

Comment:
Unfortunately, this style was done by `ocamlformat` and, we should not try to change it.

PS: the status of this `diff` is mostly because I use `ocamlformat.dev` and this patch come from after `ocamlformat.0.7`. The first one preserve parenthesis, the second one put them. To avoid conflicts in future, I prefer to let `ocamlformat` does what it wants instead to trick on styles due to the experimental status of `ocamlformat`.

Reply:
ugh, I hope someone will change that in `ocamlformat`. :)